test/e2e/apimachinery/protocol.go

```diff
@@ -34,7 +35,6 @@ import (
 
 var _ = SIGDescribe("client-go should negotiate", func() {
 	f := framework.NewDefaultFramework("protocol")
-	f.SkipNamespaceCreation = true
```

Comment:
does this create a namespace per case below?

Reply:
No, I was not sure if that was necessary, as I encode the content-type into the name of the object, so we get unique objects anyway. The original test was fine without creating any objects (this is all negotiation code we're testing). So I'm happy to create one namespace + object as a preamble as well